docs/index.rst

```diff
@@ -62,6 +62,7 @@ Table of Contents
    quickstart
    workflows/index
    bindings
+   rbac
    restapi
    changes
    tech-preview
```

Comment:
update this file too https://github.com/pulp/pulp_rpm/blob/main/docs/tech-preview.rst

Reply:
updated